cfy_manager/components/rabbitmq/rabbitmq.py

```diff
@@ -625,6 +627,7 @@ def configure(self):
             config[RABBITMQ]['ca_path'] = constants.CA_CERT_PATH
         if not config[RABBITMQ]['join_cluster']:
             self._write_definitions_file()
+        self._deploy_rebalancer_script_and_create_cronjob()
```

Comment:
Also, as you noted in slack: Yes, put this under the section where we're actually joining a cluster.